Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -413,17 +413,11 @@ - (BOOL)textInputShouldReturn
   // `onSubmitEditing` is called when "Submit" button
   // (the blue key on onscreen keyboard) did pressed
   // (no connection to any specific "submitting" process).
-#if TARGET_OS_OSX // [TODO(macOS Candidate ISS#2710739)
```

Comment:
Why don't we need this anymore?

Reply:
This slipped through. I believe we still need this.